src/esp/gfx_batch/Renderer.h

```diff
@@ -573,13 +637,47 @@ class Renderer {
    *
    * Returns a view on all node transformations in given scene. Desired usage
    * is to update only transformations at indices returned by
-   * @ref addMeshHierarchy(), updating transformations at other indices is
-   * possible but could have unintended consequences. Modifications to the
-   * transformations are taken into account in the next @ref draw().
+   * @ref addNodeHierarchy() and @ref addEmptyNode(), updating transformations
+   * at other indices is possible but could have unintended consequences.
+   * Modifications to the transformations are taken into account in the next
+   * @ref draw(). By default, transformations of root nodes (those which IDs
+   * were returned from @ref addNodeHierarchy() or @ref addEmptyNode()) are
+   * identities, transformations of other nodes are unspecified.
    */
   Corrade::Containers::StridedArrayView1D<Magnum::Matrix4> transformations(
       Magnum::UnsignedInt sceneId);
 
+  /**
+   * @brief Colors of all lights in the scene
+   * @param sceneId   Scene ID, expected to be less than @ref sceneCount()
+   *
+   * Returns a view on colors of all lights in given scene. Desired usage is to
+   * update only colors at indices returned by @ref addLight(), updating colors
+   * at other indices is possible but could have unintended consequences.
```

Comment:
Just curious, how do we end up with lights in here other than via `addLight`?

Reply:
From lights being implicitly added via `addHierarchy()`, [as discussed above](https://github.com/facebookresearch/habitat-sim/pull/1970#discussion_r1063643063).

Reply:
To be clear: is this fully hooked up? If we add a hierarchy that includes a light, will the light get added?

Reply:
Not yet -- I didn't find time to actually equip datasets with lights to have something to test with and didn't want to add completely uncovered code to this PR, so it's not here here. The "compositor" in #1969 and the glTF exporter also don't handle those for a similar reason, but both are relatively trivial pieces of code.

Once I have data to test with, I will be definitely bugging everyone on Slack with this feature ;)